Website/Packages/datax-pipeline/src/modules/flowDefinition/flowModels.js

```diff
@@ -229,6 +260,11 @@ export const outputSinkerTypes = [
         name: 'Event Hub',
         disabled: false
     },
+    {
+        key: sinkerTypeEnum.sql,
+        name: 'SQL Server',
```

Comment:
Which SQL Server does this enable exactly?

Reply:
Supports Microsoft SQL Server, Azure SQL, Azure SQL Datawarehouse. Basically anything that supports Microsoft JDBC Driver for SQL Server.